xmake/core/base/process.lua

```diff
@@ -262,6 +262,11 @@ function process.openv(program, argv, opt)
         end
     end
 
+    -- handle paths parameters
+    for k, arg in pairs(argv) do
+        argv[k] = tostring(arg)
+    end
```

Comment:
please remove it, we need only handle it for batchcmds:runv in project generator.

This will affect some performance globally and we will not deal with them for now unless necessary.

Reply:
I put this to fix the issue when xmake executes batchcmds itself:
```
lynix@SirDesktop:/mnt/c/Projets/Perso/Tests/xmake-build/test_batchcmd$ xmake.exe
[ 20%]: generating.bin2c src\xmake.png
error: ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:56: invalid argv[3] type(table) for process.openv
stack traceback:
        @programdir\core\base\utils.lua:290: in function <@programdir\core\base\utils.lua:280>
        [C]: in function 'error'
        @programdir\core\base\os.lua:854: in function 'os.raiselevel'
        (...tail calls...)
        ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:56: in field 'co_start_withopt'
        @programdir\modules\private\async\runjobs.lua:199: in function <@programdir\modules\private\async\runjobs.lua:159>
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'sandbox/modules/utils.trycall'
        @programdir\core\base\scheduler.lua:487: in function 'base/scheduler.co_group_begin'
        ...mdir\core\sandbox\modules\import\core\base\scheduler.lua:94: in field 'co_group_begin'
        @programdir\modules\private\async\runjobs.lua:159: in function <@programdir\modules\private\async\runjobs.lua:54>
        (...tail calls...)
        @programdir\actions\build\build.lua:244: in function <@programdir\actions\build\build.lua:238>
        (...tail calls...)
        @programdir\actions\build\main.lua:102: in global '_do_build'
        @programdir\actions\build\main.lua:149: in function <@programdir\actions\build\main.lua:143>
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'sandbox/modules/utils.trycall'
        @programdir\core\sandbox\modules\try.lua:121: in global 'try'
        @programdir\actions\build\main.lua:141: in function <@programdir\actions\build\main.lua:107>
        (...tail calls...)
        [C]: in function 'xpcall'
        @programdir\core\base\utils.lua:280: in function 'sandbox/modules/utils.trycall'
        (...tail calls...)
        @programdir\core\base\task.lua:519: in function 'base/task.run'
        @programdir\core\main.lua:278: in upvalue 'cotask'
        @programdir\core\base\scheduler.lua:371: in function <@programdir\core\base\scheduler.lua:368>
```

there's probably a way better place to fix this

Reply:
you can do it here

https://github.com/xmake-io/xmake/blob/166c21c46cfc79104bab7a5b6646ee43ebad45db/xmake/modules/private/utils/batchcmds.lua#L82-L110